### ISSUE TITLE
Reader doesn't read hyperlink - set_area_hyperlink and set_area_style fail

### ISSUE THREAD

**Issue by sandraros** [abap2xlsx/abap2xlsx#938]
To reproduce:
1. Run `ZDEMO_EXCEL1` and download to file `01_HelloWorld.xlsx`
2. Open `01_HelloWorld.xlsx`, see that cell B4 has an hyperlink
2. Run `ZDEMO_EXCEL37` with input file `01_HelloWorld.xlsx` and display result
3. You can see that the cell B4 has lost its hyperlink

**Comment by sandraros** [comment]
Also issue with `SET_AREA_STYLE`, it fails during `set_area_style` (error "Unable to interpret input as column"):
```abap
REPORT.
DATA: gc_save_file_name TYPE string VALUE 'issue-938-style.xlsx'.
INCLUDE zdemo_excel_outputopt_incl.
START-OF-SELECTION.
  TRY.
      DATA(lo_excel) = NEW zcl_excel( ).
      DATA(lo_worksheet) = lo_excel->get_active_worksheet( ).
      lo_worksheet->set_cell( ip_column = 'A' ip_row = 1 ip_value = 'Hello world' ).
      DATA(lo_style) = lo_excel->add_new_style( ).
      lo_style->font->color-rgb = zcl_excel_style_color=>c_red.
      lo_worksheet->set_area_style( ip_column_start = 'A' ip_row = 1 ip_style = lo_style->get_guid( ) ).
      lcl_output=>output( cl_excel = lo_excel ).
    CATCH cx_root INTO DATA(lx_root).
      MESSAGE lx_root TYPE 'I' DISPLAY LIKE 'E'.
  ENDTRY.
```

**Comment by sandraros** [comment]
I don't reproduce the error anymore. Was solved by #930. Closing and deleting the PR.
